### ISSUE TITLE
Mute and unmute for Windows?

### ISSUE THREAD

**Issue by poloma** [gillstrom/vol#11]
It seems that .set() volume when mute does not unmute on Windows. I would try and contribute but I'm not sure what the 'volume.exe' is or how it's used

**Comment by gillstrom** [comment]
I haven't touched this repo for a while so I don't remember exactly :(. But it should be from the link in this comment: https://github.com/gillstrom/vol/issues/4#issuecomment-128655453.

**Comment by all12jus** [comment]
I got an error saying that the mute function is not defined. Then I consoled out the module and it only shows the get and set functions. If you aren't going to have the mute function update the documentation?

**Comment by gillstrom** [comment]
@all12jus The documentation tells you that `mute` does only exist in the CLI part of the module.

**Comment by all12jus** [comment]
@gillstrom Please do a quick review of the npm listing for your package.  https://www.npmjs.com/package/vol

**Comment by kevva** [comment]
@all12jus, the `mute` command only occurs in the CLI section.

**Comment by NetOpWibby** [comment]
That info shouldn't be in the usage section then. Using this for a cross-platform project and am getting the same issue on macOS.

EDIT: Ah, the usage section on npm is different than the repo. Welp.

**Comment by kevva** [comment]
Yup, the newest changes aren't published to `npm` yet. If you want the docs for the stable version, check out the releases tab on GitHub.

**Comment by NetOpWibby** [comment]
That would explain the issues I was having when running examples from GH and not npm. Welp.
